File: intro_to_programming_with_ruby/7_hashes/3.rb
h = {'one': 1, 'two': 2, 'three': 3, 'four': 4}

puts "All of the keys:"
h.each_key { |x| puts x }

puts "All of the values:"
h.each_value { |x| puts x }

puts "All of the keys and values:"
h.each { |k, v| puts "#{k}: #{v}" }

File: 210/exercises/javascript_basics/3.js
if (condition1) {
  // ...(1)
  if (condition2) {
    // ...(1 && 2) one
  } else {
    // ...(1 && !2) two
  }
} else {
  // ...(!1) three
  if (condition4) {
    // ...(!1 && 4) four
    if (condition5) {
    // ...(!1 && 4 && 5) five 
    }
  }
}
